api/src/main/java/jakarta/data/repository/jpa/KeysetAwarePage.java

```diff
@@ -15,7 +15,10 @@
  *
  *  SPDX-License-Identifier: Apache-2.0
  */
-package jakarta.data.repository;
+package jakarta.data.repository.jpa;
```

Comment:
Keyset pagination is not specific to Jakarta Persistence.  It also applies to JDBC and to any database type that supports the `And` and `Or` keywords.  And in simple scenarios with only one sort criterion, it doesn't even need `Or`.

Reply:
As above, if we renamed the package name to "relational"